From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Tue, 8 Nov 2022 15:00:41 +0300
Subject: Do not configure in etc subdirectory

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f14e551..cacaed4 100644
--- a/Makefile
+++ b/Makefile
@@ -80,7 +80,7 @@ dist:
 # {{{ User level target--  config
 
 config:
-	@cd etc && $(MAKE) config   --no-print-directory
+	#@cd etc && $(MAKE) config   --no-print-directory
 	@cd lisp && $(MAKE) config --no-print-directory
 	@echo "Configured emacspeak in directory $(SRC)."
 
