From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Sat, 12 Nov 2022 12:47:57 +0300
Subject: One common rule

---
 info/Makefile | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/info/Makefile b/info/Makefile
index f0b02a7..15b8d98 100644
--- a/info/Makefile
+++ b/info/Makefile
@@ -26,12 +26,6 @@ info: emacspeak.info
 emacspeak.info: emacspeak.texi $(INCLUDES)
 	$(MAKEINFO) emacspeak.texi
 
-introducing-emacspeak.info: introducing-emacspeak.texi
-	$(MAKEINFO) introducing-emacspeak.texi	
-
-turning-twenty.info: turning-twenty.texi
-	$(MAKEINFO) turning-twenty.texi
-
 docs.texi: ../lisp/*.el
 	$(EMACS) $(BATCH) -l ../utils/self-document.el -f self-document-all-modules
 
@@ -57,6 +51,8 @@ pub:  html/index.html introducing-emacspeak.html
 	rsync introducing-emacspeak.html ${SF_HOME}
 	rsync -avz  html ${SF_HOME}/info/
 
+%.info: %.texi
+	$(MAKEINFO) $<
 
 clean: 
 	rm -f *.info* *.html *.ps *.dvi *.pdf *.txt
