From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Tue, 3 Oct 2023 15:06:09 +0300
Subject: Eliminate query on exit

---
 lisp/dtk-speak.el | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lisp/dtk-speak.el b/lisp/dtk-speak.el
index 601c125..e3b4f01 100644
--- a/lisp/dtk-speak.el
+++ b/lisp/dtk-speak.el
@@ -1646,6 +1646,7 @@ program. Port defaults to dtk-local-server-port"
       (cd "~")
       (setq process (apply #'start-process name nil program dtk-program-args)))
     (set-process-coding-system process 'utf-8 'utf-8)
+    (set-process-query-on-exit-flag process nil)
     process))
 
 (defun dtk-initialize ()
