From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Thu, 20 Aug 2026 19:34:27 +0300
Subject: Don't change speaking messages mode after search

---
 lisp/emacspeak-advice.el | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lisp/emacspeak-advice.el b/lisp/emacspeak-advice.el
index 2827069..8c11ae1 100644
--- a/lisp/emacspeak-advice.el
+++ b/lisp/emacspeak-advice.el
@@ -840,6 +840,7 @@ see option emacspeak-untabify-fixes-non-breaking-space."
        (when
            (and
             (null inhibit-message)
+            (null isearch-mode)
             m ; our message
             emacspeak-speak-messages    ; speaking messages
             (< 0.1
@@ -2078,12 +2079,10 @@ Produce an auditory icon if possible."
 (defun emacspeak-isearch-setup ()
   "Setup emacspeak environment for isearch."
   (emacspeak-auditory-icon 'open-object)
-  (setq emacspeak-speak-messages nil)
   (dtk-speak (isearch-message-prefix)))
 
 (defun emacspeak-isearch-teardown ()
   "Teardown emacspeak environment for isearch."
-  (setq emacspeak-speak-messages t)
   (emacspeak-auditory-icon 'close-object))
 
 (add-hook 'isearch-mode-hook 'emacspeak-isearch-setup)
